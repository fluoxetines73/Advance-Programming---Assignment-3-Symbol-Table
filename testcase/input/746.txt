INSERT f_FF string
ASSIGN f_FF 'abcABC'
ASSIGN __ g 123
LOOKUP f_FF
PRINT
RPRINT
LOOKUP f_FF
PRINT
LOOKUP f_FF
INSERT z number
ASSIGN f_FF 'Test123'
LOOKUP z
END
INSERT b number