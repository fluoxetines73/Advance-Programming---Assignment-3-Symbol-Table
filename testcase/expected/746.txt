Invalid: ASSIGN __ g 123